idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/utils/IdeUtils.kt

```diff
@@ -185,6 +189,41 @@ fun PsiElement.ctx(): CompilerContext? =
 fun Project.ctx(): CompilerContext? =
   getService(CompilerContext::class.java)
 
+fun <P : PsiElement> AnnotationBuilder.registerLocalFix(
+  fix: LocalQuickFix,
+  psi: P,
+  message: String,
+  highlightType: ProblemHighlightType = ProblemHighlightType.INFORMATION,
+  onThefly: Boolean = true,
+  afterBorderLine: Boolean = true
+): AnnotationBuilder.FixBuilder =
+  newLocalQuickFix(fix, psi.inspectionManager()
+    .createProblemDescriptor(psi, message, arrayOf(fix), highlightType, onThefly, afterBorderLine))
+
+fun <P : PsiElement> descriptor(
+  element: P,
+  message: String,
+  fix: LocalQuickFix? = null,
+  highlightType: ProblemHighlightType = ProblemHighlightType.INFORMATION,
+  onThefly: Boolean = true
+): ProblemDescriptor =
+  element.inspectionManager().createProblemDescriptor(element, message, fix, highlightType, onThefly)
+
+fun PsiElement.inspectionManager(): InspectionManager =
+  InspectionManager.getInstance(project)
+
+fun localQuickFix(name: String, familyName: String, f: Project.(ProblemDescriptor) -> Unit): LocalQuickFix =
+  object : LocalQuickFix {
+    override fun getName(): String =
```

Comment:
Do you know what name does?
I am only aware of the behavior of `familyName`

Reply:
I think `name` is the name of the quick fix that is going to be applied (what is displayed in the list of options).. what does the `familyName`? I set it, but it isn't shown anywhere that I'm aware of

Reply:
Could you send in a screenshot when overriding `name`?
AFAIK `familyName` is the message that is going to be displayed. When users hover over the annotation.

Reply:
No screenshot needed bud, I just deleted it because its the same 🤦 :
   /**
   * @return the name of the quick fix.
   */
  @Nls(capitalization = Nls.Capitalization.Sentence)
  @NotNull
  default String getName() {
    return getFamilyName();
  }